### ISSUE TITLE
Unable to send message using latest telegram-send package

### ISSUE THREAD

**Issue by achaknalwar** [rahiel/telegram-send#121]
```
$ telegram-send --version
telegram-send 0.34
```

```
$ pip show python-telegram-bot
Name: python-telegram-bot
Version: 20.1
Summary: We have made you a wrapper you can't refuse
Home-page: https://python-telegram-bot.org/
Author: Leandro Toledo
Author-email: devs@python-telegram-bot.org
License: LGPLv3
Location: /user/lib/python3.9/site-packages
Requires: httpx
Required-by: telegram-send
```

```
$ telegram-send "Hi"
/user/lib/python3.9/site-packages/telegram_send/telegram_send.py:247: RuntimeWarning: coroutine 'Bot.send_message' was never awaited
  message_ids += [send_message(m, parse_mode)["message_id"]]
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
Traceback (most recent call last):
  File "/user/bin/telegram-send", line 8, in <module>
    sys.exit(main())
  File "/user/lib/python3.9/site-packages/telegram_send/telegram_send.py", line 126, in main
    message_ids += send(
  File "/user/lib/python3.9/site-packages/telegram_send/telegram_send.py", line 247, in send
    message_ids += [send_message(m, parse_mode)["message_id"]]
TypeError: 'coroutine' object is not subscriptable
```

**Comment by JohSchillerEmnify** [comment]
This is due to how telegram-send uses the coroutine  in line 243 and line 247: 
```python
message_ids += [send_message(m, parse_mode)["message_id"]] 
```
Because send_message is a coroutine object it cannot be subscripted directly so using ["message_id"] will throw this exception. It would either have to be implemented with asyncio or some other way, further down in the code, it is just left out. 

Not sure why this pops up now, possibly the telegram bot package got an update.

**Comment by JohSchillerEmnify** [comment]
The owner of this repo seems to largely have abandoned this project. So either you implement your own version or you look to one of the forks to see if one of them is still actively developed

**Comment by rahiel** [comment]
This is fixed in version 0.35, please upgrade.

> The owner of this repo seems to largely have abandoned this project. So either you implement your own version or you look to one of the forks to see if one of them is still actively developed

Forks are welcome to contribute upstream :)
